inst/include/interface/rcpp/rcpp_interface.hpp

```diff
@@ -174,6 +174,22 @@ std::string finalize_fims(Rcpp::NumericVector par, Rcpp::Function fn,
   return ret;
 }
 
+/**
+ * @brief Sets the fixed parameters vector object.
+ *
+ * @param par A vector of parameter values.
+ */
```

Comment:
Thank you for documenting this parameter and the next one. Unfortunately I still don't know what they do or when I would use it. Can you add some more information?